pkg/util/encoding/encoding.go

```diff
@@ -81,8 +81,9 @@ const (
 	bitArrayDataTerminator     = 0x00
 	bitArrayDataDescTerminator = 0xff
 
-	timeTZMarker = bitArrayDescMarker + 1
-	geoMarker    = timeTZMarker + 1
+	timeTZMarker  = bitArrayDescMarker + 1
+	geoMarker     = timeTZMarker + 1
```

Comment:
I'm not sure why you need this -- it looks like you're just serializing geos to bytes, and then encoding the bytes. Same for decoding -- we'll always know that we have some geo's when decoding the bytes.

Reply:
i'd rather differentiate for now - especially as we're not (fully) decided on the encoding scheme for geo in protobuf form.

Reply:
OK, that makes sense then.